BLEDiscovery.cpp for loop with uninitialized i in 1.5.0 release

### Operating System

Windows 10

### IDE version

2.1.0

### Board

Feather nRF52840 Express

### BSP version

1.5.0

### Sketch

Any sketch using Bluefruit52 lib

### What happened ?

It appears the recent commit, Add LOG_LV2 for BLE Characteristic UUIDs on Discovery, forgot to initialize i in the for loop at line 243. Not sure if this breaks anything functionally (I'm not familiar with what `LOG_LV2()` does), but it does spit out a compiler warning. 

### How to reproduce ?

Compile any sketch using the Bluefruit52 library (`#include <bluefruit.h>`) with compiler warnings set to "More" or "All"

### Debug Log

```
\Arduino15\packages\adafruit\hardware\nrf52\1.5.0\libraries\Bluefruit52Lib\src\BLEDiscovery.cpp: In member function 'void BLEDiscovery::_eventHandler(ble_evt_t*)':
\Arduino15\packages\adafruit\hardware\nrf52\1.5.0\libraries\Bluefruit52Lib\src\BLEDiscovery.cpp:243:23: warning: 'i' may be used uninitialized in this function [-Wmaybe-uninitialized]
  243 |       for (uint8_t i; i < chr_rsp->count; i++) {
      |                       ^
```

### Screenshots

_No response_

--- a/libraries/Bluefruit52Lib/src/BLEDiscovery.cpp
+++ b/libraries/Bluefruit52Lib/src/BLEDiscovery.cpp
@@ -240,7 +240,7 @@ void BLEDiscovery::_eventHandler(ble_evt_t* evt)
 
       LOG_LV2("DISC", "[CHR] Characteristic Count: %d", chr_rsp->count);
 
-      for (uint8_t i; i < chr_rsp->count; i++) {
+      for (uint8_t i=0; i < chr_rsp->count; i++) {
         LOG_LV2("DISC", "[CHR] Characteristic #%d: 0x%04X", i, chr_rsp->chars[i].uuid.uuid);
       }
 

